### ISSUE TITLE
onNodeRightClick doesnt on macbook keyboard+trackpad

### ISSUE THREAD

**Issue by eunukasiko** [vasturiano/force-graph#350]
**Describe the bug**
onNodeRightClick is not called

**To Reproduce**
1. Hold control key
2. Press the trackpad button

**Expected behavior**
onNodeRightClick should be called

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] macOS 13.6.6 (22G630)
 - Browser [e.g. chrome, safari] Chrome  Version 124.0.6367.156 (Official Build) (x86_64)
 - Version [e.g. 22] 1.43.5

**Comment by vasturiano** [comment]
@eunukasiko I don't believe the browser interprets a trackpad press + ctrl key as a right button click event. That's a translation that possibly happens elsewhere. Does the `onNodeClick` function get called instead? That's the expected behavior.

**Comment by eunukasiko** [comment]
Yes, I ended up writing a special case for onNodeClick.
